### ISSUE TITLE
Herobrine Crashes The Game Upon Placing a Sign

### ISSUE THREAD

**Issue by Trklntry** [Jo5629/jo5629-In_the_Fog#1]
I'm playing multiplayer with someone and the game's continuously crashing, his stalking and such works but he's struggling with signs. Signslib is also installed, previously there was a different error when it wasn't installed. Here's the error message:
```
AsyncErr: Lua: Runtime error from mod 'herobrine' in callback environment_Step(): /usr/local/share/minetest/builtin/common/misc_helpers.lua:422: bad argument #2 to 'format' (number expected, got nil)
stack traceback:
        [C]: in function 'format'
        /usr/local/share/minetest/builtin/common/misc_helpers.lua:422: in function 'pos_to_string'
        ...est/mods/in_the_fog/herobrine/functions/random_signs.lua:75: in function 'place_sign'
        ...est/mods/in_the_fog/herobrine/functions/random_signs.lua:136: in function <...est/mods/in_the_fog/herobrine/functions/random_signs.lua:132>
        /usr/local/share/minetest/builtin/common/register.lua:26: in function </usr/local/share/minetest/builtin/common/register.lua:12>

```

**Comment by Jo5629** [comment]
Oversight on my end sorry.

**Comment by Jo5629** [comment]
Bug should be fixed in v1.2.1
